### DISCUSSION TITLE
Is possible to buy copilot for other user?

### DISCUSSION THREAD

**Discussion by Fover23** [community/community#33004] [Copilot Conversations]
I went to buy copilot for other team menbers that without credit card .

**Comment by iamcjwilliams** [comment]
@Fover23 Can you send a screenshot of the error you're receiving?

**Reply by Fover23** [reply]
I already subscribe copilot for myself. Can I change my subscription for organization or create new subscription for other team menbers?

**Reply by iamcjwilliams** [reply]
@Fover23 Copilot is only offered for individuals at this time, you will need to create another subscription for other team members. Please visit here for more [information](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription).

**Reply by Fover23** [reply]
Thank you.
